### PR TITLE
OSDOCS#5402: Updated IBM CLoud to IBM Cloud Bare Metal (Classic)

### PR DESCRIPTION
**Pull request by dfitzmau**
[OSDOCS-5402](https://issues.redhat.com/browse/OSDOCS-5402)

Version(s):
4.14 through to 4.10

Link to docs preview:
* [IBM Cloud Bare Metal (Classic)](https://61782--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_ibm_cloud/install-ibm-cloud-prerequisites.html)
* [ROSA docs - Understanding the DNS Operator](https://61782--docspreview.netlify.app/openshift-rosa/latest/networking/dns-operator.html#nw-dns-forward_dns-operator)

QE review:
- [x] QE has approved this change.

Additional information:
* [Slack confirmation that IBM Cloud on VPC does not require name updates](https://redhat-internal.slack.com/archives/C04T569EL1Z/p1687965689808089)
* [Slack in IBM channel](https://redhat-internal.slack.com/archives/C01QK05L93R/p1688118414866959)
* [IBM doc that states OCP support](https://www.ibm.com/blog/openshift-version-4-13-now-available-in-red-hat-openshift-on-ibm-cloud/)

### CODE REVIEW

File: _distro_map.yml [hunk 1]
```diff
@@ -195,16 +195,6 @@ openshift-rosa-portal:
     enterprise-4.13:
       name: ''
       dir: rosa-portal/
-partner-roks:
```
Comment (kalexand-rh) [inline _distro_map.yml:198]: @wgordon17, I don't think this distro is in use anymore. Do you have any concerns about removing it?

**Comment by yithian** [comment]
The DNS changes look sane to me. Thanks!

File: _distro_map.yml (hunk 1 above)
Comment (kalexand-rh) [reply _distro_map.yml:198]: @dfitzmau, we're trying to figure out what's going on with this distro. The [docs](https://docs.openshift.com/roks/4/welcome/index.html) exist, even though they're small and, to my knowledge, unmaintained. For now, please restore the distro section and correct the name instead.

Comment (dfitzmau) [reply _distro_map.yml:198]: Thanks, @kalexand-rh . I restored the original file's content. I'm not sure if I need to change the name as I'm uncertain what part of our doc set this file applies.

Maybe https://docs.openshift.com/roks/4/welcome/index.html but I'm not sure how the file's entries would render to this name unless something add "name+(sub)name+site_name" parameters were bundles together by some backend file.

Comment (kalexand-rh) [reply _distro_map.yml:198]: @dfitzmau, my apologies. Will you delete it again?

Comment (dfitzmau) [reply _distro_map.yml:198]: Yes, NP. Removed the section from the `_distro` file.

Comment (dfitzmau) [reply _distro_map.yml:198]: Hi @kalexand-rh . I do not see the doc book for `OpenShift on IBM Cloud` when I navigate to the root directory of the preview link. Maybe removing the entry from the `_distro` YAML file removed this doc book? I have been informed that the title of this doc book is wrong: `Red Hat OpenShift on IBM Cloud 4 Documentation` > `Red Hat OpenShift 4 on IBM Cloud Documentation`. We can drop the `4` also, so the book is a bit more future-proof (if we need to keep the doc book.

**Comment by mffiedler** [comment]
@gpei @sunzhaohua2 PTAL

**Comment by gpei** [comment]
@MayXuQQ could you please take a look on this?

**Comment by sgoveas** [comment]
[IBM Cloud Bare Metal (Classic)](https://61782--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_ibm_cloud/install-ibm-cloud-prerequisites.html) looks good to me

File: _attributes/common-attributes.adoc [hunk 2]
```diff
@@ -146,6 +146,9 @@ endif::[]
 :SMProductVersion1x: 1.1.18.2
 //Windows containers
 :productwinc: Red Hat OpenShift support for Windows Containers
+// IBM Cloud
+:ibmcloudBMProductName: IBM Cloud Bare Metal (Classic)
+:ibmcloudBMRegProductName: IBM Cloud&#174; Bare Metal (Classic)
```
Comment (lahinson) [inline _attributes/common-attributes.adoc:151]: Just curious -- are we required to include the registered trademark in the first instance of "IBM Cloud"? I hadn't seen that before in Red Hat documentation, so I'm just asking for my own knowledge.

**Review by lahinson — COMMENTED** [review]
LGTM! This is pretty straightforward, and I didn't have any revisions to suggest. I did have one question about the inclusion of the registered trademark.

File: _attributes/common-attributes.adoc (hunk 2 above)
Comment (dfitzmau) [reply _attributes/common-attributes.adoc:151]: Hi @lahinson . Timing could not have been better: see, https://github.com/redhat-documentation/supplementary-style-guide/pull/359#discussion_r1265446103

**Comment by jab-rh** [comment]
@bergerhoffer, I don't think this should happen?

**Comment by bergerhoffer** [comment]
@jab-rh thanks for the ping! You're right that PRs shouldn't be failing to 4.14.

It looks like it's failing on the distro map. I can see from the history that there are several changes that went into the distro map in main but not 4.14. But I'm not familiar enough with the distro map to know if changes are only supposed to be made against main for it or if they're also supposed to go into the enterprise branches. @kalexand-rh can you help with how that's supposed to work?

**Comment by kalexand-rh** [comment]
None of the distros that the CP is trying to add are based off the 4.14 branch, so the map was never updated to include them. IMO, it's ok for `main` and `enterprise-4.14` to diverge here, and we should merge the manual CP PR that I made while checking out the conflict.

**Comment by bergerhoffer** [comment]
Thanks, merged it

**Comment by bergerhoffer** [comment]
@dfitzmau That was the 4.14 one, but we still need ones for 4.10-4.13 too, if you can take care of those please!

**Pull request merged**